contracts/express/AxelarExpressExecutable.sol

```diff
@@ -12,12 +12,12 @@ contract AxelarExpressExecutable is ExpressExecutorTracker {
     using SafeTokenTransfer for IERC20;
```

Comment:
Try inheriting from `AxelarGMPExecutableWithToken`. Make `execute` and `executeWithToken` virtual, so they can be overridden in this method, and _execute and _executeWithToken don't need to be redefined. Also makes it so that commandId is passed in to those methods